src/engine/ruleset/integrations/system/decoders/system-auth.yml

```diff
@@ -139,7 +139,7 @@ normalize:
       - "new group: name=<group.name>, GID=<group.id>"
       - "new user: name=<user.name>, UID=<user.id>, GID=<group.id>, home=<_system.auth.useradd.home>, shell=<_system.auth.useradd.shell>(?,<~>)"
       - "group added to <file.name>: name=<group.name>(?, GID=<group.id>)"
```

Comment:
Should the double quotes be removed too?

Reply:
in this case not since they enclose: which is reserved by yaml